docs/static/resources/openapi.json

```diff
@@ -17912,7 +17912,7 @@
                       "description": "Menu items in a forest like data structure",
                       "items": {
                         "properties": {
-                          "childs": {
+                          "children": {
                             "items": {
                               "type": "object"
```

Comment:
These look scary, but I think you fixed the corresponding logic, so we're all good here.